help/move-to-zulip-cloud.md

```diff
@@ -0,0 +1,135 @@
+# Move from self-hosting to Zulip Cloud
+
+With Zulip's high quality import and export tools, you can always
+move from self-hosting your own Zulip server to using the Zulip
+Cloud service (and back).
+
+## Process overview
+
+To move your Zulip organization from a self-hosted server to Zulip
+Cloud, you will need to take the following steps, which are described
+in more detail below:
+
+{start_tabs}
+
+1. [Plan the process and coordinate with Zulip support](#plan-the-process-and-coordinate-with-zulip-support).
+
+1. [Upgrade your self-hosted server](#upgrade-your-self-hosted-server).
+
+1. [Export organization data](#export-organization-data).
+
+1. [Decide how users will log in](#decide-how-users-will-log-in).
+
+{end_tabs}
+
+## Plan the process and coordinate with Zulip support
+
+To import your self-hosted organization into Zulip Cloud, your server will need
+to have the same database version as Zulip Cloud. Zulip Cloud is updated every
+couple of weeks, so it's important to coordinate the timing with Zulip's support
+team.
+
+{start_tabs}
+
+1. Review the process described on this page, and decide when you will be ready
+   to make the transition.
+
+1. [Create a new Zulip Cloud organization](https://zulip.com/new/)
+   with the subdomain you would like for your Zulip organization. Your
+   Zulip Cloud organization will be hosted at `<subdomain>.zulipchat.com`.
+
+1. Email [support@zulip.com](mailto:support@zulip.com) with the following information:
+      - URL of your new Zulip Cloud organization
+      - Your estimated timeline for generating a data export
+      - Any other timing considerations for the transition (e.g., time of day)
+
+Zulip's support team will coordinate with you to make the transition with
+minimal disruption for your team.
+
+{end_tabs}
+
+## Upgrade your self-hosted server
+
+You will likely need to upgrade your server to have the same database version as
+Zulip Cloud. Zulip Cloud runs the `zulip-cloud-current` branch plus some
+cherry-picked changes.
+
+{start_tabs}
+
+1. [Check](/help/view-zulip-version#view-zulip-server-and-web-app-version) your
+   Zulip server version.
+
+1. [Upgrade to the latest maintenance
+   release](https://zulip.readthedocs.io/en/stable/production/upgrade.html#upgrading-to-a-release)
+   if you are running an older version of the Zulip server.
+
+1. [Upgrade](https://zulip.readthedocs.io/en/stable/production/upgrade.html#upgrading-from-a-git-repository)
+   to the `zulip-cloud-current` branch.
+
+{end_tabs}
+
+For additional support with upgrading from an older version of Zulip, contact
+[sales@zulip.com](mailto:sales@zulip.com) for paid support options.
+
+## Export organization data
+
+{start_tabs}
+
+1. Shut down access to your Zulip organization.
```

Comment:
We have that already in https://zulip.readthedocs.io/en/stable/production/export-and-import.html#preventing-changes-during-the-export, which is linked in the next section. I think maybe "announce the migration and scheduled downtime for your Zulip server to your team" would be a better fit here.

Reply:
OK. I had not idea what this part was about. :)

Reply:
Forgot to comment that we should explain what you need to do, but cutting sounds fine given that.